testing/unit-testing-jest/docs/getting-dom-elements.md

```diff
@@ -0,0 +1,63 @@
+
+# How to get elements from the DOM on testing
+Check if an element render the correct information on the page is very important to test and more if we are frontend developers, so to test the elements you need to access to the elements and also call his events.
+
+## Getting element
+Consider the next element
+test.html
+```html
+<button class="btn_login" (click)="login()">Login</button>
+```
+
+test.spec.ts
+```ts
+it('should call login function', () => {
+    // as nativeElement
+    const button = fixture.debugElement.nativeELement.querySelector('.btn_login');
+    button.click();
+    
+    ...
+    
+    // as debugElement
+    const button = fixture.debugElement.query(By.css('.btn_login'));
+    button.triggerEventHandler('click', null);
```

Comment:
which is the purpose of the second argument?

Reply:
The event object expected by the handler. But, we could avoid send it.